browser-support-molecule.css

```diff
@@ -118,15 +117,61 @@ body {
 }
 
 .cs__main--examples .example__container:nth-of-type(1) {
-  margin-top: 0px;
+  margin-top: 0;
 }
 
 /* CODE BOX STYLE */
 
-.codebox {
+.cb {
+  min-width: 500px;
   width: 100%;
-  min-height: 300px;
+}
+
+.cb__buttons {
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  margin-bottom: 20px;
+}
+
+.cb__buttons__button {
   border: 1px solid #3CDAF2;
+  border-radius: 4px;
+  color: #3CDAF2;
+  font-size: 18px;
+  font-weight: 500;
+  margin-right: 6px;
+  padding: 0.5em 1em;
+  position: relative;
+  text-align: center;
+  width: 150px;
+}
+
+.cb__buttons__button--active {
+  background-color: #3CDAF2;
+  color: #F4F9FC;
+}
+
+.cb__buttons__button--active::after {
+  border-left: 10px solid transparent;
+  border-right: 10px solid transparent;
+  border-top: 10px solid #3CDAF2;
+  content: '';
+  height: 0;
+  left: 65px;
+  position: absolute;
+  top: 100%;
+  width: 0;
+}
+
+.cb__code {
+  background-color: #F4F9FC;
+  border: 2px solid #3CDAF2;
+  border-radius: 4px;
+  font-size: 20px;
+  font-weight: 300;
+  padding: 30px;
+  white-space: pre;
```

Comment:
Line 174 also kills the responsiveness of the page. You should use `white-space: pre-wrap` instead. So that the width stay intact as the viewport resizes.

Also it would be really good to wrap the code block inside a `pre` and `code` tags, as these tags are semantic. Refer to this answer in Stackoverflow.
http://stackoverflow.com/a/8306643/4947178